share/scripts/breakpoints.tcl

```diff
@@ -0,0 +1,41 @@
+# Easily save breakpoints to and restore from a file
+
+proc save_breakpoints_help {args} {
+	return {save_breakpoints [<filename>]
+
+Save all breakpoints to a given file.
+}}
+
+proc save_breakpoints {filename} {
+	set all [debug breakpoint list]
+	set fh [open $filename "w"]
+	puts $fh [list {*}$all]
+	close $fh
+}
+set_help_proc save_breakpoints save_breakpoints_help
+
+proc load_breakpoints_help {args} {
+	return {load_breakpoints [<filename>]
+
+Load only new breakpoints from a given file, while preserving pre-existing ones if any.
+}}
+
+proc load_breakpoints {filename} {
+	set fh [open $filename "r"]
+	set all [gets $fh]
+	close $fh
+	# compare and create breakpoints if not found
+	foreach {_ data1} [eval list $all] {
```

Comment:
We completely ignore the name of the saved breakpoint. That's fine. But if we don't need that info, then maybe we also shouldn't save it?

I think with `[dict values ...]` you can ignore the keys from a key-value-pair list.

Caveat: I haven't actually tried any of this.